vehicles/src/modules/auth/jwt.strategy.ts

```diff
@@ -112,8 +113,7 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
           : companyId;
 
         if (
-          !associatedCompanies.includes(companyId) ||
-          associatedCompanyMetadataList?.at(0)?.isSuspended
+          !associatedCompanies.includes(companyId)
         ) {
```

Comment:
removed the `isSuspended` guard here, since we're gonna need to display to the User specific messaging

Reply:
you should not be removing the guard

Reply:
Removing this check is a blunder. Reveals more information than necessary. You should instead use the user context response and add a field to onroutebc context for is suspended.

Reply:
makes more sense thanks 🙇